app/components/events/view/export/download-zip.js

```diff
@@ -1,4 +1,27 @@
 import Component from '@ember/component';
+import { action } from '@ember/object';
 
-export default Component.extend({
-});
+export default class extends Component {
+    @action
+  exportEventDownload(eventDownloadUrl) {
+    this.set('isLoading', true);
+    this.loader
+      .load(`${eventDownloadUrl}`, { withoutPrefix: true })
+      .then(res => {
+        const anchor = document.createElement('a');
+        anchor.style.display = 'none';
+        anchor.href = URL.createObjectURL(new Blob([res], { type: 'octet/stream' }));
+        anchor.download = 'EventExport.zip';
+        anchor.click();
```

Comment:
Does this work on all browsers ? Please test on 
- Chrome
- FF
- IE 11
- Edge
- Mobile Chrome
- Mobile Safari 
- Safari

Reply:
Tested on Chrome, Safari, Firefox. Works fine.
I've also tested it on Mobile Safari(iPhone) & Mobile Chrome too. Works alright.